addons/sale_project/models/project_project.py

```diff
@@ -394,44 +394,32 @@ def _get_sale_order_items_query(self, domain_per_model=None):
 
     def get_panel_data(self):
         panel_data = super().get_panel_data()
-        if self.allow_billable:
-            sale_items = self.get_sale_items_data(limit=5)
-            revenues = panel_data['profitability_items']['revenues']['data']
-            for section in revenues:
-                if section['id'] in sale_items:
-                    section['sale_items'] = sale_items[section['id']]['data']
-                    section['displayLoadMore'] = sale_items[section['id']]['displayLoadMore']
-                    section['isFolded'] = True
+        foldable_sections = self._get_foldable_section()
+        if self._show_profitability() and 'revenues' in panel_data['profitability_items']:
+            for section in panel_data['profitability_items']['revenues']['data']:
+                if section['id'] in foldable_sections:
+                    section['isSectionFoldable'] = True
```

Comment:
Cannot we set that in the JS code? 🧐

Reply:
I did it on a previous iteration, but vincent thought it was cleaner to handle it on the backend side.
https://github.com/odoo/odoo/pull/179286#discussion_r1751501889

Reply:
Okay 🙂 

let's keep as you did then. 😁